lib/chef/provisioning/convergence_strategy/ignore_convergence_failure.rb

```diff
@@ -0,0 +1,53 @@
+class Chef
+module Provisioning
+  class ConvergenceStrategy
+
+    # The purpose of this class is to decore the `converge` method with logic to catch any
+    # convergence failure exceptions, log them and then squelch them.  The reason we
+    # need this is to prevent 1 provisioned node's converge failing an entire provisioning
+    # recipe.
+    module IgnoreConvergenceFailure
+
+      attr_accessor :ignore_failures_array, :ignore_exit_values
+
+      # This module is only meant to be extended into instances, not classes or modules
```

Comment:
It might be useful to have a comment explaining why.  It isn't clear to me why this style of composition is desirable here.